app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { NavLink } from 'redux-first-router-link';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import { activeProjectSelector } from 'controllers/user';
+import {
+  activeDashboardIdSelector,
+  PROJECT_DASHBOARD_PAGE,
+  PROJECT_DASHBOARD_ITEM_PAGE,
+} from 'controllers/pages';
+import { dashboardItemsSelector } from 'controllers/dashboard';
+import { InputDropdown } from 'components/inputs/inputDropdown';
+import { AddDashboardButton } from './addDashboardButton';
+import styles from './dashboardPageHeader.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  allDashboardsTitle: {
+    id: 'DashboardPageHeader.allDashboardsTitle',
+    defaultMessage: 'All dashboards',
+  },
+});
+
+const DASHBOARD_PAGE_ITEM_VALUE = 'All';
+
+@connect((state) => ({
+  projectId: activeProjectSelector(state),
+  dashboards: dashboardItemsSelector(state),
+  activeItemId: activeDashboardIdSelector(state),
+}))
+@injectIntl
+export class DashboardPageHeader extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    projectId: PropTypes.string.isRequired,
+    activeItemId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    dashboards: PropTypes.array,
```

Comment:
Let's make it easier to understand what we expect here:
`PropTypes.arrayOf(Prop.Types.shape(...))`

Reply:
Fixed